File: RadioJavan/Classes/Models/RadioJavanRegister.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace RadioJavan.Classes.Models
{
    public class RadioJavanRegister
    {
        [JsonPropertyName("success")]
        public bool Success { get; set; }
        [JsonPropertyName("redirect")]
        public string Redirect { get; set; }
        [JsonPropertyName("user_id")]
        public string UserId { get; set; }
        [JsonPropertyName("msg")]
        public string Message { get; set; }
    }
}

File: Dart/lib/src/Classes/Models/RadioJavanRegister.dart
class RadioJavanRegister {
  bool Success;
  String Redirect;
  String UserId;
  String Message;

  RadioJavanRegister.fromJson(Map<String, dynamic> json)
      : Success = json['success'],
        Redirect = json['redirect'],
        UserId = json['user_id'],
        Message = json['msg'];
}
